crates/tools/js/benchmark/package.json

```diff
@@ -14,9 +14,9 @@
   "keywords": [],
   "author": "",
   "license": "ISC",
-  "dependencies": {
+  "devDependencies": {
     "argparse": "^2.0.1",
-    "hardhat": "^2.23.0-dev.0",
+    "hardhat": "*",
```

Comment:
Specify Hardhat version at the workspace level

Reply:
Are you sure that the semantic of `*` is "use the version at the worksapce level"? I always interpreted it as "use the latest dependency". Right now that matches the version in the root because that is the latest version, but I'm not sure that will always be the case.

If you want to guarantee that we are using the same version everywhere, we could adapt the [`check-dependencies`](https://github.com/NomicFoundation/hardhat/blob/main/scripts/check-dependencies.js) script from Hardhat and run it in the CI.

Reply:
Uhh good catch this is what I get for asking ChatGPT how to use workspace level dependency definition 😬 

Fixed in: [Use specific version of Hardhat](https://github.com/NomicFoundation/edr/pull/507/commits/6f7a27d76a1070204c62c6394bf9da4c026f3f33)